packages/esm-patient-forms-app/translations/fr.json

```diff
@@ -1,6 +1,6 @@
 {
   "all": "Tout",
-  "clinicalForm": "Formulaire clinique",
+  "clinicalForms": "Formulaire clinique",
```

Comment:
I see a change to `clinicalForms` here and in the Cambodian translations, but not in the English. These two should probably be undone... Otherwise, we need to have the translations re-checked (the French for "Clinical forms" should be something like "Formulaires clinique"; I have no idea what change or even if a change is needed in Khmer)